cranelift/codegen/benches/x64-evex-encoding.rs

```diff
@@ -0,0 +1,111 @@
+use cranelift_codegen::isa::x64::encoding::{
```

Comment:
I think this entire benchmark should be feature-gated (i.e., `#[cfg(feature = "x86")]`) but I don't know the best way to do this?

Reply:
You could put the whole implementation inside an inner `mod { ... }` with the #[cfg(...)]`, export one main function, and have an alternate (empty) main function with `[cfg(not(feature = "x86"))]` probably?

Reply:
Ok, I finally figured out how to do this. My latest update uses the inner module approach feature-gated on `x86` and prints a "can't run this benchmark" message if the feature is not present.